apiary.apib

```diff
@@ -3444,6 +3449,25 @@ Delete a task.
 
 + Response 204
 
+
+### tasks.schedule [POST /tasks.schedule]
+
+Schedule a task.
```

Comment:
```suggestion
Schedule a task in your calendar.
```

Reply:
PP: I think adding "in your calendar" is confusing, I don't think we have the concept of calendar today in the API.